### PR TITLE
Seeder after fix DB

### PR DESCRIPTION
**Pull request by annq3012**
## Description
Add and change dataseeder for all of table
## Reference Requirements
- Product Backlog ID: PBI-0
- Task ID: #0.3

## Migrations
YES 

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

File: database/factories/ModelFactory.php [hunk 1]
```diff
@@ -18,6 +18,22 @@
...
+});
+
+$factory->define(App\Model\Donator::class, function (Faker $faker) {
+    return [
+    ];
+});
+
 $factory->define(App\Model\Book::class, function (Faker $faker) {
     return [
         'QRcode' => $faker->ean13,
```
Comment (at-anhngo) [inline database/factories/ModelFactory.php]: chỗ này có thể chỉnh lại thành BAT-xxxxx cho đúng định dạng không?

**Review by ghost — DISMISSED** [review]
Check my comment

File: database/factories/ModelFactory.php [hunk 2]
```diff
@@ -18,9 +18,24 @@
...
 });
 
+$factory->define(App\Model\User::class, function (Faker $faker) {
+    $team = ['PHP', 'SA', 'QC', 'Adroid', 'IOS'];
+    return [
+        'employee_code' => 'AT-' . $faker->unique()->randomNumber(3),
+        'name'                  => $faker->name,
+        'email'                  => $faker->safeEmail,
+        'team'                   => $team[array_rand($team)],
+        'role'                     => rand(0, 1),
```
Comment (at-xinhnguyen) [inline database/factories/ModelFactory.php]: format lại cho dễ nhìn tí An ơi, 
các dấu => thẳng hàng với nhau ấy

Comment (at-annguyen) [reply database/factories/ModelFactory.php]: ok rồi anh :+1:

**Comment by at-duyho** [comment]
- pleas, cross review before assign to me. thank you!

**Comment by at-duyho** [comment]
-But, I didn't found his approvement.

**Comment by at-anhngo** [comment]
LGTM

File: database/factories/ModelFactory.php [hunk 3]
```diff
@@ -18,32 +18,46 @@
...
+        'image'        => $faker->image,
+        'status'       => $faker->numberBetween($min = 0, $max = 1),
     ];
 });
 
 $factory->define(App\Model\Borrowing::class, function (Faker $faker) {
     return [
         'from_date' => $faker->datetime,
-        'to_date' => $faker->datetime,
+        'to_date'   => $faker->datetime,
```
Comment (at-duyho) [inline database/factories/ModelFactory.php]: to_date should after from_date. You shoud use: to_date = from_date + constant.

File: database/factories/ModelFactory.php [hunk 4]
```diff
@@ -56,6 +70,19 @@
 
 $factory->define(App\Model\Favorite::class, function (Faker $faker) {
     return [
+        'favoritable_id'   => rand(1,15),
+        'favoritable_type' => $faker->randomElement(['books', 'comments', 'posts'])
```
Comment (at-duyho) [inline database/factories/ModelFactory.php]: - please check values of favoritable_type? normally, values is a namespace of class.

**Review by ghost — CHANGES_REQUESTED** [review]
Please, check my comment

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**